matching/dist/controllers/matching-controller.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
```

Comment:
I'm assuming this is the compiled code? Am thinking we can include `dist/` in the `.gitignore` file